R/format_PEW_temp.R

```diff
@@ -0,0 +1,419 @@
+#'                 HatchDateD0 = as.Date(HatchDateD0),
+#'                 NumberOfRingedChicks = as.integer(NumberOfRingedChicks),
+#'                 DateRingingChicks = as.Date(DateRingingChicks),
+#'                 NoOfChicksD3 = as.integer(NoOfChicksD3),
+#'                 BroodMassD3 = as.numeric(BroodMassD3),
+#'                 ObservationTimeH = as.numeric(ObservationTimeH),
+#'                 Seks = substr(Seks, 1, 1),
+#'                 Species = "CYACAE",
+#'                 PopID = "PEW") %>%
+#'   #### Rename variables to standardized format
+#'   dplyr::rename(BreedingSeason = Year,
+#'                 IndvID = Id,
+#'                 Sex =  Seks,
+#'                 NestboxID = Nest,
+#'                 ObserverID = Measurer) %>%
+#'   #### Remove columns which we do not store in the standardized format
+#'   dplyr::select(-FeatherCollection ,
+#'                 -BreathRateTime50Breaths,
+#'                 -FeathersPartner,
+#'                 -BreathRatePartnerTime50Breaths,
+#'                 -BloodSample,
+#'                 -TimeBloodSample,
+#'                 -BloodSampleDuration,
+#'                 # -NewRing,
+#'                 -VisitRateVisitsH,
+#'                 -VrPartner,
+#'                 -VisitsAlternated,
+#'                 -VisitsAlternatedPartner,
+#'                 -VisitsSync10,
+#'                 -ChickAgeOfBehavObserv,
+#'                 -MateStrategy) %>%
+#'   #### Reorder columns
+#'   dplyr::select(BreedingSeason,
+#'                 Species,
+#'                 PopID,
+#'                 IndvID,
+#'                 Sex,
+#'                 Age,
+#'                 everything())
+#'
+#'
+#'
+#'
+#' #### check columns with NAs
+#' # sort(apply(pew_data, 2, function(x) sum(is.na(x))), decreasing = TRUE)
+#'
+#'
+#'
+#' #### --------------------------------------~
+#' #### INDIVIDUAL DATA                    #### done, need to check
+#' #### --------------------------------------~
+#'
+#' #### Still to solve:
+#' # There are some individuals (for example 13617043), where the rows of data related
+#' # to the same individual seem complementary, but are in two rows with
+#' # insufficient information when treating the rows separately.
+#' # How to deal with that?
+#' # >>> So far, I remove duplicated rows at the end of the function,
+#' # but there are some rows still maintained, as there is some difference in one
+#' # of the columns of the dataframe.
+#' #
+#' # There is no data about broodID, so far I used NA.
+#' #
+#' # Remove data with no date in the primary data?
+#'
+#'
+#'
+#' message("Compiling individual information...")
+#'
+#' Individual_data <- create_individual_PEW(pew_data)
+#'   dplyr::filter(Species %in% species)
+#'
+#'
+#' create_individual_PEW <- function(data){
+#'
+#'   Individual_data <-
+#'     data %>%
+#'     #### Remove rows without bird ID
+#'     filter(!is.na(IndvID)) %>%
+#'     #### CHECK: Remove rows without date
+#'     # filter(!is.na(Date)) %>%
```

Comment:
See comment above, I think this is correct of Capture_data (and corresponding Individual_data) 👍 😄

Reply:
ok